### PR TITLE
[WIP] list social media accounts

### LINKED ISSUE #125: List Social Media Accounts
As a User
i`d like to enter my social media accounts 
in order to show them to other users to get in contact

### PR DESCRIPTION
**Pull request by Kachulio1**
fix #125

### CODE REVIEW

**Comment by roschaefer** [comment]
@Kachulio1 a build was canceled + branch needs to be updated

https://dashboard.cypress.io/#/projects/qa7fe2/runs/1147/specs looks nice! How many social media accounts can we add?

**Comment by appinteractive** [comment]
Should we limit it to max 5 accounts?

**Comment by roschaefer** [comment]
@appinteractive nah, I was like: "Cool, I didn't try out the code but I'm curious how many accounts can we add?" I would limit the accounts and fix a problem that we don't have at the moment.

File: cypress/integration/08.SocialMediaAccounts.feature [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+  Background:
+    Given I have a user account
+    And I am logged in
+    And I am on the "settings" page
+
+  Scenario: Adding Social Media Acccounts
+    Given I click on the "My social media accounts" link
+    Then I should be on the "/settings/my-social-media-accounts" page
+    And I should be able to add a social media link
```
Comment (roschaefer) [inline cypress/integration/08.SocialMediaAccounts.feature]: @Kachulio1 could you add another assertion to see the social media account link on the page? The [video](https://dashboard.cypress.io/#/projects/qa7fe2/runs/1147/specs) stops before we can see how the link appears.

Comment (mattwr18) [reply cypress/integration/08.SocialMediaAccounts.feature]: at the moment, the link doesn't appear, we are still working on that implementation

**Comment by roschaefer** [comment]
Closed in favour of https://github.com/Human-Connection/Human-Connection/pull/237

**Pull request closed without merging**